plugins/devtools/README.md

```diff
@@ -456,3 +456,37 @@ RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
     apt-get update && \
     apt-get install -y  ... iputils-ping
 ```
+
+### Scheduled Tasks Configuration
+
+Scheduled tasks can be viewed and triggered in the DevTools plugin under the `Scheduled Tasks` tab. You first must add the list of plugins for scheduled tasks to your config:
+
+```yaml
+devTools:
+  scheduledTasks:
+    plugins:
+      - catalog
+```
+
+Then add a new `DevToolsLayout.Route` to the end of your `DevToolsLayout` in your `CustomDevToolsPage.tsx` like this:
```

Comment:
I think this should be included out-of-the-box as part of the default DevTools tab so this won't be needed. Can you also make the changes so it is part of the default tabs?